src/px-script/sheet/index.js

```diff
@@ -11,6 +11,7 @@ class Sheet {
         this.id = el.id;
         this.closeIcon = el.querySelector(SELECTORS.CLOSEICON);
         this.isOpen = el.classList.contains("sheet-open");
+        this.hasVScroll = () => window.innerWidth - document.documentElement.clientWidth;
```

Comment:
Is this supposed to return a negative number if it has a scrollbar and a positive if not?

Either rename to something like `scrollbarWidth` or change to something like
```suggestion
        this.hasVScroll = () => (window.innerWidth - document.documentElement.clientWidth) > 0;
```